grimoire_elk/enriched/mediawiki.py

```diff
@@ -219,7 +222,10 @@ def get_rich_item(self, item):
         if "revisions" in page:
             eitem["nrevisions"] = len(page["revisions"])
             if len(page["revisions"]) > 0:
-                eitem["first_editor"] = page["revisions"][0]["user"]
+                if 'user' in page["revisions"][0]:
```

Comment:
Shouldn't this be better?

```suggestion
                page["revisions"][0].get('user', HIDDEN_EDITOR)
```